### DISCUSSION TITLE
availability in masterfile locales.json is missing

### DISCUSSION THREAD

**Discussion by topdev-spetermann** [requarks/wiki#6010] [Help / Questions] [answered]
We have a setup with kubernetes where external communication is not allowed. So our solution for localization was to use the offline mode and build a custom image from the base Wiki.js image, where we preload the localization files during the build process of the image. This way works fine, but in the admin ui we can not see the availability of each locale because this information would normally be loaded from the GraphQL endpoint which is of course blocked in our environment. The problem is, that the field "availability" is not included in the locales.json - File in the repository [requarks/wiki-localization](https://github.com/requarks/wiki-localization).
Our approach currently loads the localization files at the build time of the docker image from the GraphQL endpoint, but maybe it's simpler to add the field "availability" to the nightly export? I would make a pull request, but i didn't found the workflow, who does this job and update the repository. Can you help us?

**Comment by NGPixel** [comment, marked as answer]
Upgrade to 2.5.295.

**Comment by topdev-spetermann** [comment]
I made the pull request, so i'm already at 2.5.295. The point is that the nigthly export of locales.json at [requarks/wiki-localization](https://github.com/requarks/wiki-localization) doesn't contain the information for availability. Look here: https://github.com/requarks/wiki-localization/blob/master/locales.json

**Reply by NGPixel** [reply]
Fixed.
